### PR TITLE
Fix additional Python and JupyterLab edge case

### PR DESCRIPTION
**Pull request by jungaretti**
Related: https://github.com/devcontainers/images/issues/125

Fixes an edge case in the Python feature that installs JupyterLab to the `additionalVersions` of Python instead of the primary `version` of Python.

This PR also removes junk from our JupyterLab test and adds a new test that would have caught this bug.

### Edge Case Details

`install_user_package` installs packages to whatever Python installation is specified by `$INSTALL_PATH`. Both `install_from_source` and `install_using_oryx` overwrite this variable. When we use `install_python` to install an extra version of Python, we overwrite `$INSTALL_PATH` to that installation.

When we install JupyterLab at the end of the script, it uses the `$INSTALL_PATH` set by the most recent `additionalVersions` instead of `version`.

### CODE REVIEW

File: test/python/scenarios.json [hunk 1]
```diff
@@ -12,15 +12,24 @@
...
-                "username": "vscode"
-            },
             "python": {
                 "installJupyterlab": true,
                 "configureJupyterlabAllowOrigin": "*"
             }
         }
     },
+    "install_additional_jupyterlab": {
+        "image": "mcr.microsoft.com/vscode/devcontainers/base:focal",
```
Comment (samruddhikhandale) [inline test/python/scenarios.json:22]:
```suggestion
        "image": "mcr.microsoft.com/devcontainers/base:focal",
```

Comment (samruddhikhandale) [reply test/python/scenarios.json:22]: Can we use the new image paths?

**Review by samruddhikhandale — APPROVED** [review]
LGTM ✨ 
Can we make the image path change before merging this PR?

**Review by samruddhikhandale — DISMISSED** [review]
Thanks!

**Review by samruddhikhandale — APPROVED** [review]
